src/features/student/views/StudentToolsTracesView/components/StudentToolsTracesViewNotice/StudentToolsTracesViewNotice.vue

```diff
@@ -33,11 +37,11 @@ const maxDayBeforeDeletion = computed(() => tracesConfig.value?.maxRemainingDays
 
 <template>
   <div
-    v-if="unassignedTracesSummary && unassignedTracesSummary.total > 0"
+    v-if="unassignedTracesSummary && unassignedTracesSummary?.unassociated > 0"
```

Comment:
```suggestion
    v-if="unassignedTracesSummary && unassignedTracesSummary.unassociated > 0"
```

Grâce au if normalement t'as pas besoin de ?